lib/dpl/provider/pages.rb

```diff
@@ -58,7 +58,7 @@ def github_deploy
         context.shell "git config user.email '#{@gh_email}'"
         context.shell "git config user.name '#{@gh_name}'"
         context.shell "echo '#{@gh_fqdn}' > CNAME" if @gh_fqdn
-        context.shell 'git add .'
+        context.shell 'git add . > /dev/null'
```

Comment:
I think you're right. I think the problem I'm seeing is actually on the commit, e.g. https://travis-ci.org/drush-io/help/builds/254300569#L177